tests/unit/test_workflows.py

```diff
@@ -101,12 +101,15 @@ def test_to_dict(self, sig_factory):
         w1.add_dependency(w2)
         w1.add_dependency(w3)
 
+        w1.custom_payload["foo"] = "bar"
+
         d = w1.to_dict()
 
         assert d['id'] == 'task-1'
         assert d['dependencies']
         assert d['dependencies']['task-2'] is None
         assert d['dependencies']['task-3'] is None
+        assert d['custom_payload']['foo'] is "bar"
```

Comment:
I know it works, but it looks awkward :)
```
assert d['custom_payload']['foo'] == "bar"
```

Reply:
No problem to fix it, but just fyi, it's used several times throughout that file (e.g. lines 113, 166, 167)